### PR TITLE
ISC - Restructure

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

Branch to preview the changes from all issues in https://github.com/iotaledger/iota-wiki/issues/1359

## Links to any relevant issues

fixes https://github.com/iotaledger/iota-wiki/issues/1359

## Type of change

- Documentation Enhancement

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [ ] I have performed a self-review of my own changes
- [ ] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by jorgemmsilva — COMMENTED** [review]
These sections need to be revisited and written in the context of "ISC contracts" (core/wasm), these texts don't apply the same way to EVM contract (EVM contracts for example don't need to use the ISC sandbox to access their state). That needs to be clear.:
- Sandbox Interface
- Calling a Smart Contract


The entire "Explanations" section I think has too much focus, most people don't need to know any of that to create EVM contracts (which is what we are prioritizing). I think a "call to action" from the introduction, directly to the "How To's" would be good.


The "How To's" additions are really good in general!

File: docs/build/isc/v1.0.0-rc.6/docs/getting-started/compatibility.md [hunk 1]
```diff
@@ -33,14 +33,14 @@ Here are some of the most important properties and limitations of EVM support in
 
 ### Wrapped Calls to the JSON-RPC
 
-The Wasp node provides a JSON-RPC service, the standard protocol used by Ethereum tools. Upon receiving a signed
+The Wasp node provides a JSON-RPC service, the standard protocol used by Ethereum tools. We try to keep our endpoint as compatible as possible, you can find a list of supported endpoints [here](../reference/json-rpc-spec.md). Upon receiving a signed
```
[the remark this answers is not in the document]
Comment (lucas-tortora) [reply docs/build/isc/v1.0.0-rc.6/docs/getting-started/compatibility.md:36]: This has already been addressed in another PR https://github.com/iotaledger/iota-wiki/pull/1509 , so the restructure doesn't block the update.

File: docs/build/isc/v1.0.0-rc.6/docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+supports [EVM/Solidity](getting-started/languages-and-vms.md#evmsolidity-based-smart-contracts)
+smart contracts, as well as an **experimental** [Wasm VM](getting-started/languages-and-vms.md#wasm-vm-for-isc).
+
+## Sandbox Interface
+
+Smart contracts access data via the deterministic [Sandbox interface](explanations/sandbox.md), ensuring
+security and predictability. This interface restricts contracts to their own state and provides various utilities like cryptographic functions and
+event dispatching.
+
+![Sandbox](/img/sandbox.png)
```
[the remark this answers is not in the document]
Comment (lucas-tortora) [reply docs/build/isc/v1.0.0-rc.6/docs/introduction.md:59]: We've created another issue to track these https://github.com/iotaledger/devx/issues/342

File: docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx [hunk 3]
```diff
@@ -27,50 +32,53 @@ directly with an IOTA Smart Contracts chain running EVM as long as you take a co
 1. Please make sure you use the correct JSON-RPC endpoint URL in your tooling for your chain. You can find the JSON-RPC
    endpoint URL in the Wasp dashboard (`<URL>/wasp/dashboard` when using `node-docker-setup`).
 2. Please ensure you use the correct `Chain ID` configured while starting the JSON-RPC service. If you did not
-   explicitly define this while starting the service, the default Chain ID will be `1073`.
+   explicitly define this while starting the service, the default Chain ID will be
+**<ChainId url='https://json-rpc.evm.shimmer.network'/>** for ShimmerEVM or
+**<ChainId url='https://json-rpc.evm.testnet.shimmer.network'/>** for the ShimmerEVM Testnet .
```
[the remark this answers is not in the document]
Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx]: The chainID component will automatically pull the chain if from the json rpc. You can look at it in the preview:
https://iota-wiki-r5sphch8c-iota1.vercel.app/isc/getting-started/tools/#tooling-considerations
Bildschirmfoto 2024-03-11 um 11 05 48

[the remark this answers is not in the document]
Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx]: This section has some weird formatting. That's why there is a missing space after the chain ID. I will update everything to use markdown only.

[the remark this answers is not in the document]
Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx]: Done https://github.com/iotaledger/iota-wiki/pull/1458/commits/f4142be8bc518e2f9b98908334fa241b08106a62

File: docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/basics/get-balance.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+import "@iota/iscmagic/ISC.sol";
+
+contract GetBalance {
+    event GotAgentID(bytes agentID);
+    event GotBaseBalance(uint64 baseBalance);
+    event GotNativeTokenBalance(uint256 nativeTokenBalance);
+    event GotNFTIDs(uint256 nftBalance)
+
+    function getBalance(bytes nativeTokenID) public {
```
Comment (Dr-Electron) [inline docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/basics/get-balance.md:58]:
```suggestion
    event GotNFTIDs(uint256 nftBalance);

    function getBalance(bytes memory nativeTokenID) public {
```

**Pull request merged**